src/hardware/adc.rs

```diff
@@ -75,34 +119,46 @@ impl Adc {
         pins.cs.2.set_high().unwrap();
         pins.cs.3.set_high().unwrap();
 
-        // SPI at 1 MHz. SPI MODE_0: idle low, capture on first transition
-        let spi: Spi<_, _, u8> = spi4.spi(spi_pins, spi::MODE_0, 12500.khz(), spi4_rec, clocks);
+        // set sync low first for synchronization at rising edge
+        pins.sync.set_low().unwrap();
 
-        let bus_manager = shared_bus_rtic::new!(spi, Spi<SPI4, Enabled>);
+        // SPI MODE_3: idle high, capture on second transition
+        let spi: Spi<_, _, u8> = spi4.spi(pins.spi, spi::MODE_3, 12500.khz(), spi4_rec, clocks);
 
         let mut adc = Adc {
-            adcs: (
-                ad7172::Ad7172::new(delay, bus_manager.acquire(), pins.cs.0).unwrap(),
-                ad7172::Ad7172::new(delay, bus_manager.acquire(), pins.cs.1).unwrap(),
-                ad7172::Ad7172::new(delay, bus_manager.acquire(), pins.cs.2).unwrap(),
-                ad7172::Ad7172::new(delay, bus_manager.acquire(), pins.cs.3).unwrap(),
-            ),
+            adcs: ad7172::Ad7172::new(spi),
+            rdyn: pins.rdyn,
+            sync: pins.sync,
+            cs: pins.cs,
+            current_position: 0,
         };
 
-        Adc::setup_adc(&mut adc.adcs.0);
-        Adc::setup_adc(&mut adc.adcs.1);
-        Adc::setup_adc(&mut adc.adcs.2);
-        Adc::setup_adc(&mut adc.adcs.3);
+        setup_adc!(adc, delay, 0);
+        setup_adc!(adc, delay, 1);
+        setup_adc!(adc, delay, 2);
+        setup_adc!(adc, delay, 3);
+
+        // set sync high after initialization of all phys
+        // TODO: double check timing after last setup and generally more datasheet studying for this
+        adc.sync.set_high().unwrap();
 
         adc
     }
 
     /// Setup an adc on Thermostat-EEM.
```

Comment:
Please be consistent with your wording, spelling and capitalization (or reduce it). Is it `ADC` or `adc`.